### PR TITLE
Upgrade terraform-provider-aws to v5.62.0

### PR DESCRIPTION
**Pull request by flostadler**
- **Moving upstream to v5.62.0**
- **./scripts/tidy-all.sh**
- **./scripts/patch_computed_only.sh**
- **Fix TODOs in replacements.json and add test guarding against forgotten TODOs**
    - We had some older TODOs in `replacements.json`, I fixed those up as well and added a test so we don't forget again
- **Regenerate schema**
- **Regenerate SDK**

### CODE REVIEW

File: patches/0054-Fix-empty-retry_strategy-of-Batch-JobDefinitions-cau.patch [hunk 1]
```diff
@@ -1,32 +0,0 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```
Comment (t0yv0) [inline patches/0054-Fix-empty-retry_strategy-of-Batch-JobDefinitions-cau.patch:1]: This patch was redundant?

Comment (flostadler) [reply patches/0054-Fix-empty-retry_strategy-of-Batch-JobDefinitions-cau.patch:1]: My upstream PR finally got merged in https://github.com/hashicorp/terraform-provider-aws/pull/37500

**Review by t0yv0 — APPROVED** [review]
LGTM 🚢

**Comment by mikhailshilkov** [comment]
This PR has been shipped in [release v6.49.0](https://github.com/pulumi/pulumi-aws/releases/tag/v6.49.0).

**Pull request merged**